src/machine/board_particle_argon.go

```diff
@@ -94,3 +94,14 @@ const (
 	NFC1_PIN          Pin = 9
 	NFC2_PIN          Pin = 10
 )
+
+// USB CDC identifiers
+const (
+	usb_STRING_PRODUCT      = "Particle Argon"
+	usb_STRING_MANUFACTURER = "Particle"
+)
+
+var (
+	usb_VID uint16 = 0x239A
+	usb_PID uint16 = 0x8029
```

Comment:
This is the VID from Adafruit. Doesn't Particle have a different VID? (CC @suda).

Reply:
The Adafruit drivers use the Adafruit VID for boards that do not have their own, such as Particle:
https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/boards.txt#L444

Reply:
Yep, Particle has different PIDs for all three boards. I wanted to add them in a separate PR after this one is merged in 👍🏻

Reply:
The VID for 3rd gen is `0x2B04` and PIDs are respectively: `0xD00C` for Argon, `0xD00D` for Boron and `0xD00E` for Xenon ([source](https://github.com/particle-iot/device-os/blob/master/build/platform-id.mk#L287)).

Reply:
I propose after we merge this PR then @suda can create a separate PR to connect the ids for the Particle boards, as he mentioned.

Reply:
This PR was merged, so @suda we are ready for your separate PR.

Reply:
@deadprogram https://github.com/tinygo-org/tinygo/pull/985 done!